src/components/Guilds/ProposalPage/ProposalVoteCard/index.tsx

```diff
@@ -174,9 +234,9 @@ const ProposalVoteCard = () => {
         isOpen={modalOpen}
         onDismiss={() => setModalOpen(false)}
         onConfirm={confirmVoteProposal}
-        selectedAction="Yes"
-        votingPower={0.12}
-        totalLocked={0}
+        selectedAction={selectedAction?.toString() === '1' ? 'Yes' : 'No'}

```

Comment:
For now lets name this the same as it is on the voting sidebar, we can have more than 2 votes

Reply:
Sure.